src/libraries/System.Net.WebSockets.Client/src/System/Net/WebSockets/WebSocketHandle.Managed.cs

```diff
@@ -257,21 +257,23 @@ private static WebSocketDeflateOptions ParseDeflateOptions(ReadOnlySpan<char> ex
 
                 if (value.Length > 0)
                 {
-                    if (value.SequenceEqual(ClientWebSocketDeflateConstants.ClientNoContextTakeover))
+                    switch (value)
                     {
-                        options.ClientContextTakeover = false;
-                    }
-                    else if (value.SequenceEqual(ClientWebSocketDeflateConstants.ServerNoContextTakeover))
-                    {
-                        options.ServerContextTakeover = false;
-                    }
-                    else if (value.StartsWith(ClientWebSocketDeflateConstants.ClientMaxWindowBits))
```

Comment:
These are `StartsWith`, not `SequenceEqual`.

Reply:
Ugh, so they are.

Reply:
And my order of System.Security.*/... to the end and then get "the file I skipped over" reveals a flaw...